node/utils/event.ts

```diff
@@ -4,3 +4,4 @@ export const skuChanged = eventName('sku')
 export const productChanged = eventName('product')
 export const brandChanged = eventName('brand')
 export const categoryChanged = eventName('category')
+export const notification = eventName('notification')
```

Comment:
why not using a more descriptive name, like `broadcaster notification`, or `rawNotification`

Reply:
The name is: `broadcaster.notification`